### PR TITLE
misc: update changelog for v6.0.0

### PR DESCRIPTION
**Pull request by brendankenny**
way better than #10804

### CODE REVIEW

**Comment by paulirish** [comment]
https://github.com/GoogleChrome/lighthouse/compare/v600..changelog

File: changelog.md [hunk 1]
```diff
@@ -1,422 +1,271 @@
...
+* A PWA [`maskable-icon`](https://web.dev/maskable-icon-audit/) just looks better on your homescreen, so this new audit encourages you to have at least one available in your manifest ([#10370](https://github.com/GoogleChrome/lighthouse/pull/10370)).
+* `timing-budget` expands budget assertions to now be settable on all the performance metrics ([#9901](https://github.com/GoogleChrome/lighthouse/pull/9901), [#9925](https://github.com/GoogleChrome/lighthouse/pull/9925)).
+* The new [`charset`](https://web.dev/charset) audit ensures a proper character encoding for page content ([#10284](https://github.com/GoogleChrome/lighthouse/pull/10284), [#10389](https://github.com/GoogleChrome/lighthouse/pull/10389), [#10689](https://github.com/GoogleChrome/lighthouse/pull/10689)).
+* `image-size-responsive` checks that images have an aspect ratio and resolution that match well with how they are displaed on a page ([#10460](https://github.com/GoogleChrome/lighthouse/pull/10460)).
```
Comment (patrickhulce) [inline changelog.md:21]:
```suggestion
* `image-size-responsive` checks that images have an aspect ratio and resolution that match well with how they are displayed on a page ([#10460](https://github.com/GoogleChrome/lighthouse/pull/10460)).
```

**Comment by jayaddison** [comment]
Congratulations on the 6.0.0 release! :tada: 

Selfishly-reporting: might there be an issue with the collection of contributor handles?  (perhaps counting (contributions.length == 1) rather than than (contributions.since_prev_release > 0), or something like that?)

**Comment by patrickhulce** [comment]
Oh no! @jayaddison I'm not sure how you got cutoff I'm so sorry! 😢 

This is the script that generates the contibutors

```bash
git --no-pager shortlog -s -e -n "v2.3.0..v${OLD_VERSION}" | cut -f 2 | sort > auto_contribs_prior_to_last
git --no-pager shortlog -s -e -n "v${OLD_VERSION}..HEAD" | cut -f 2 | sort > auto_contribs_since_last
NEW_CONTRIBUTORS=$(comm -13 auto_contribs_prior_to_last auto_contribs_since_last)
```

When I run it with 5.6.0 as the `OLD_VERSION` I get you in the list, so I'm not sure what happened. Perhaps there was something squirrely with the fact that we published a 6.0.0-beta in between? 😕 

<details>
<summary>Output of Above Command with Emails Redacted</summary>


Alex Tkachuk <REDACTED>
AndreasKubasa <REDACTED>
Ben Schwarz <REDACTED>
Carr.z.w <REDACTED>
Catalin Rosu <REDACTED>
Elizabeth Sweeny <REDACTED>
François Beaufort <REDACTED>
James Addison <REDACTED>
James Addison <REDACTED>
Jan Nicklas <REDACTED>
John Sampson <REDACTED>
Kiko Beats <REDACTED>
Maisam Somji <REDACTED>
Markus Staab <REDACTED>
Mathias Bynens <REDACTED>
Matt Jared <REDACTED>
Michael Blasingame <REDACTED>
Michael Dijkstra <REDACTED>
Peter Müller <REDACTED>
Piotr Zarycki <REDACTED>
Robert Linder <REDACTED>
Roelf-Jan de Vries <REDACTED>
Roland Burrows <REDACTED>
Sebastian Kreft <REDACTED>
SphinxKnight <REDACTED>
Tim Giles Jr <REDACTED>
Umar Hansa <REDACTED>
Warren Maresca <REDACTED>
Warren Maresca <REDACTED>
Web Design Hertfordshire <REDACTED>
baseeee <REDACTED>
jazyan <REDACTED>
jazyan <REDACTED>
kmanuel <REDACTED>
nxqamar <REDACTED>
ruxin <REDACTED>
⭐ B3none <REDACTED>

</details>

**Comment by jayaddison** [comment]
No worries!  I make that 34 unique contributors, and the changelog mentions 29 (please check my math), so there may have been a few folks left out accidentally.

**Comment by brendankenny** [comment]
Arg, really sorry @jayaddison! That's my fault. Something got lost mapping from shortlog -> commit -> github username.

> No worries! I make that 34 unique contributors, and the changelog mentions 29 (please check my math), so there may have been a few folks left out accidentally.

I lost one other, but there are also three old-time committers in that list (benschwarz (#2355), beaufortfrancois (#1032), and umaar (all the way back to #150!)).

**Comment by jayaddison** [comment]
All good, thank you for the swift fix :)

**Comment by connorjclark** [comment]
> I lost one other, but there are also three old-time committers in that list (benschwarz (#2355), beaufortfrancois (#1032), and umaar (all the way back to #150!)).


maybe emails changed? if so, perhaps adding them to `.mailmap` will prevent.

**Pull request merged**